test/metabase/models/collection_test.clj

```diff
@@ -1556,4 +1568,34 @@
                              :name     "a"
                              :location "/3/"
                              :children []}]}]
-               (collection/collections->tree collections)))))))
+               (collection/collections->tree {} collections)))))))
+
+(deftest annotate-collections-test
+  (let [collections [{:id 1, :name "a", :location "/"}
+                     {:id 2, :name "b", :location "/1/"}
+                     {:id 3, :name "c", :location "/1/2/"}
+                     {:id 4, :name "d", :location "/1/2/3/"}
+                     {:id 5, :name "e", :location "/1/"}]
+        clean      #(walk/prewalk
```

Comment:
nit: I think one more space before the `#` ?

Reply:
actually this is properly aligned. The maps visible here are inside of a vector so they are indented one space for the `[` which is in the same column as the `#` for the function literal